### ISSUE TITLE
different image sizes

### ISSUE THREAD

**Issue by NorthSummer** [Megvii-BaseDetection/BEVStereo#15]
Hi, thanks for opening access.
By the way, where should I modify if I want to train with 1408x512 images? I only modified the "final_dims" and the results seem to be not right. Does the depth_gt need to be generated again when changing the input image size?

**Comment by yukaizhou** [comment]
Have you solved this problem?

**Comment by NorthSummer** [comment]
> 

Not yet, I trained BEVDepth official code with 1408x512 resolution a few days ago, but the results seemed like overfit (14.2 total loss, 29.4 NDS). However it's not reasonable...

**Comment by yukaizhou** [comment]
I also encountered a similar problem. Only the size of the input image was modified for the official code, without any other changes, and the result was very abnormal. Regarding what you said about regenerating the point cloud ground truth, I confirm that the code and image size are irrelevant. The loss value of my training eventually oscillates between 8.2 and 9.2. Do you think this is normal?
